### ISSUE TITLE
Thaumatorium as dungeon loot

### ISSUE THREAD

**Issue by squillis** [FTBTeam/FTB-Interactions#1147]
#710  Modpack Version: 
2.0.6

## Bug Report:
You can find thaumatorium as dungeon loot, but if you place them you cannot use the top half of the block, it only lets you interact with the bottom half. Which prevents you from using the emptying essentia tranfusers since you cannot click on the top half to place the block. (this can also be tested with spawned in thaumatorium, it has the same result as the looted ones.)

## Expected Behavior/Suggestions:
Stated above.

## Is It Repeatable? Steps to Reproduce: 
N/A

## Link to Log or Crash File Paste**: 
N/A

## Mod/s Affected: 
Thaumcraft

## Known Fix: 
N/A

**Comment by Saereth** [comment]
yeah because its actually only 1 block, if you break them you get 1 block etc etc. will probably replace them with just the unformed block
